build/typings/index.d.ts

```diff
@@ -70,7 +71,7 @@ declare namespace dashjs {
         on(type: StreamInitializedEvent['type'], listener: (e: StreamInitializedEvent) => void, scope?: object): void;
         on(type: TextTracksAddedEvent['type'], listener: (e: TextTracksAddedEvent) => void, scope?: object): void;
         on(type: string, listener: (e: Event) => void, scope?: object): void;
-        off(type: string, listener: (e: Event) => void, scope?: object): void;
+        off(type: string, listener: (e: any) => void, scope?: object): void;
```

Comment:
`e: Event` causes compile compile errors when typescript's `strictFunctionTypes` compiler option is enabled. So unfortunately we must use `e: any`.